styles.css

```diff
@@ -0,0 +1,69 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 80px 0 45px 0;
```

Comment:
Just to mention the alternative, have a look at the (quite new CSS property) `margin-block` ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block)): we could remove the `0` values; but you can also leave it here if you think it's possible that you need to adjust the `inline` values soon.